static/src/stylesheets/module/preferences/_notifications.scss

```diff
@@ -1,46 +1,61 @@
-@mixin notifcations-button-style($foreground: $neutral-2, $background: #ffffff) {
-    .notifications__button {
-        color: $foreground;
-        border-color: $foreground;
-        border-style: solid;
-
-        .inline-icon {
-            fill: $foreground
+@mixin notifcations-button-style($foreground: $live-main-2, $background: #ffffff) {
+    .notifications__toggle {
+        border: 0;
+        padding: 0;
+        display: block;
+        background-color: transparent;
+
+        &:hover,
+        &:focus {
+            outline: 0;
         }
     }
+}
 
-    .notifications__button--subscribed {
-        color: $background;
-        background-color: $foreground;
-        .inline-icon {
-            fill: $background;
-            background-color: $foreground;
+$slider-height: 32px;
+$slider-clearence: 3px;
+
+.live-notifications__slider {
+    float: left;
+    position: relative;
+    height: $slider-height;
+    width: $gs-gutter * 2.5;
+    border-radius: $slider-height / 2;
+    margin-right: $gs-gutter / 2;
+    transition: background .3s ease;
+
+    &:before {
+        content: '';
+        position: absolute;
+        top: $slider-clearence;
+        width: $slider-height - ($slider-clearence * 2);
+        height: $slider-height - ($slider-clearence * 2);
+        background-color: $neutral-8;
+        border-radius: 50%;
+        transition: all .3s ease;
+    }
+
+    &--unsubscribed {
```

Comment:
We try not to use `&` on BEM elements and modifications as it makes it difficult to grep the files - it would be better to un-nest this and use the full class.